### ISSUE TITLE
Dwell time sum-up rounding

### ISSUE THREAD

**Issue by jhelgert** [adbuerger/pycombina#14]
Hi there,

currently, CombinaSUR does not support dwell times, i.e. `min_up_times` and `max_down_times` are ignored. However, the dwell time sum-up rounding algorithm (DSUR) described in [this paper (section 6.1)](https://link.springer.com/article/10.1007/s10107-020-01533-x) yields promising results for huge CIA instances (n_oc * n_t > 5000), where the branch and bound algorithm is computationally too expensive. 

Are there already any plans to include the DSUR algorithm? Provided there's interest, I could submit a PR.

**Comment by r-barnes** [comment]
@jhelgert : You have an implementation for DSUR available?

**Comment by jhelgert** [comment]
@r-barnes Yes, [pyCIAP](https://github.com/jhelgert/pyCIAP) contains a straightforward implementation of the algorithm written in Cython.
